lua/ide/lib/git/client.lua

```diff
@@ -31,10 +31,9 @@ Git.new = function()
     local function handle_req(callback)
         return function(req)
             if req.error then
-                return
-            end
-            if req.exit_code ~= 0 then
-                vim.notify(string.format("git: %s", req.stderr), vim.log.levels.ERROR)
+                vim.notify(string.format("%s\n%s", req.reason, req.stderr),  "error", {
```

Comment:
@mrjones2014 - wanted to get your opinion here.

I was trying to use "vim.log.levels.*" here instead of the word "error". nvim-notify docs say it will support it, but no matter what, it always just displayed a normal status notification window. 

Am I crazy?

Reply:
Hmm, that should work. Unless the values of the enum changed?

Reply:
Maybe its my setup? I even try with just a integer, no luck.

Reply:
I can clone the branch and test it out tomorrow.